### DISCUSSION TITLE
Powered by... Thank you!

### DISCUSSION THREAD

**Discussion by frank-bxl** [linux-surface/linux-surface#618] [Show and tell]
Thank you for your developing work! 
The linux-surface project gave the power of linux to the Surface Go - a device which is everything else than powerful with it's native OS. Since the cameras got to get working too (latest kernel 15.14.14-3), the "little Go" is ready to go... 

![IMG_20211030_140856](https://user-images.githubusercontent.com/49043732/139532702-7f57a83a-1769-4dc5-94f9-b2407d1d6560.jpg)

**Comment by Tim024** [comment]
I second this post (and need the same sticker), thank you!! When I got a new job and was given a SL3... You cannot imagine the relief when I saw all this work on porting linux to Surface laptops. I am unable to work with windows products and this project is a lifesaver. Thank you again for your great work!

**Comment by frank-bxl** [comment]
![Debian_logo](https://user-images.githubusercontent.com/49043732/140653297-a413c696-4ec4-4975-85a9-9ea7d26cfe83.png)
Hello Tim,
thank you for your feedback.
The sticker is this png printed on semi transparent foil with a standard laser printer.

**Comment by dnj0104** [comment]
Hello! I just wanna join the choir, and say thanks for this amazing project , first I installed a regular debian 13 image couple months ago, but that did not worked as supposed, and today I installed this kernel for my surface 3, and it work perfect, working wifi,touch , zoom and scroll it's Magick . 
![IMG20260222185108](https://github.com/user-attachments/assets/0f3a321f-2aaf-4e3e-8fbc-97fd52d899ba)
![IMG20260222185045](https://github.com/user-attachments/assets/0e260652-ef85-4b22-b487-76de2f820254)
